docs/docs/security-in-gatsby.md

```diff
@@ -1,9 +1,161 @@
+
+```js
+// highlight-next-line
+<body onload="document.csrf.submit()">
+<!-- ... -->
+<form action="http://mywebsite.com/blog/addcomment" method="POST" name="csrf">
+  // highlight-next-line
+  <input type="hidden" name="comment" value="Hey visit http://maliciouswebsite.com, it's pretty nice" />
+  <input type="submit" />
+</form>
+```
+
+### **How can you prevent cross-site request forgery?**
+
+#### Don't use GET requests to modify data
+
+Actions that do not simply read data should be handled in a POST request. In the example above, if the `/blog/addcomment` accepts a GET request, the CSRF attack can be done using a `<img />` tag:
+
+```html
+<img src="http://mywebsite.com/blog/addcomment?comment=unwanted%20comment" />
+```
+
+#### CSRF Tokens
+
+If you want to protect a page your server will provide an encrypted, hard to guess **token**. That token is tied to a user's session and is must be included in every POST request. See the following example:
+
+```js
+<form action="http://mywebsite.com/blog/addcoment" method="POST">
+  <input type="text" name="comment" />
+  // highlight-next-line
+  <input type="hidden" name="token" value="SU9J3tMoT1w5q6uJ1VMXaaf9UXzLvyNd" />
+  <input type="submit" />
+</form>
+```
+
+When the form is sent, the server will compare the token received with the stored token and block the action if they are not the same. Make sure that malicious websites don't have access to the CSRF token by using [HTTP Access Control](https://developer.mozilla.org/pt-BR/docs/Web/HTTP/Headers/Access-Control-Allow-Origin).
+
+#### Same-Site Cookies Attribute
+
+This cookie attribute is targeted to prevent CSRF attacks. If you need to create a cookie in your application, make sure to protect them by this attribute, that could be of `Strict` or `Lax` type:
+
+`Set-Cookie: example=1; SameSite=Strict`
+
+It allows the server to make sure that the cookies are not being sent by a **cross-site** domain request.
+Check out [MDN Docs](https://developer.mozilla.org/pt-BR/docs/Web/HTTP/Headers/Set-Cookie) for more information on configuring a cookie. You will also want to note current browser support which is available on the [Can I Use page](https://caniuse.com/#feat=same-site-cookie-attribute).
+
+Quoting [OWASP](https://cheatsheetseries.owasp.org/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.html#samesite-cookie-attribute):
+
+> It is important to note that this attribute should be implemented as an additional layer defense in depth concept. This attribute protects the user through the browsers supporting it, and it contains as well 2 ways to bypass it as mentioned in the following section. This attribute should not replace having a CSRF Token. Instead, it should co-exist with that token in order to protect the user in a more robust way. Also remember that any Cross-Site Scripting (XSS) can be used to defeat all CSRF mitigation techniques!
```

Comment:
I'm wondering if we need this full quote? Is the goal to explain that you need to do this in addition to the CSRF token? If so, perhaps it should be mentioned in that section?

Reply:
The goal is to explain that the `SameSite` cookies attribute is not a replacement for CSRF Token (and vice-versa) and they can work to together as security layers in a website. Also, if there is Cross-Site Scripting (XSS) attack, both of them are useless to a CSRF attack. Is this the best place to put this information? Should it stay as a quote or I should write a new paragraph?

Reply:
I think what you just wrote here is clearer. So perhaps a paragraph and then a link to that information if they want to read more?

Reply:
Yes, makes sense. I replaced this quote with a paragraph, adding the link as well.

Reply:
The new text is no longer a quote, correct?

Reply:
correct!